src/main.rs

```diff
@@ -18,9 +18,13 @@ fn main() {
     };
 
     let result = ContextBuilder::new("Moon & Sun", "wedyarit")
-            .window_mode(conf::WindowMode::default().dimensions(1280.0, 1280.0))
+            .window_mode(conf::WindowMode::default()
+                .maximized(true)
```

Comment:
I assume there is no need to set the default maximum window size. Window size should be comfort.